core/src/epicli/cli/epicli.py

```diff
@@ -18,7 +19,10 @@ def main():
         formatter_class=argparse.RawDescriptionHelpFormatter)
 
     # setup some root arguments
-    parser.add_argument('--version', action='version', version=VERSION)
+    parser.add_argument('--version', action='version', help='Shows the CLI version', version=VERSION)
+    parser.add_argument('--licenses', action='version',
```

Comment:
action='version' same as for --version, is it correct?

Reply:
Yes the version action just returns a string so Im using that to just return the license info under a different command. No need to make it more complicated:)